app/components/RelatedRecordTableRow.tsx

```diff
@@ -61,68 +81,141 @@ export default function RelatedRecordTableRow<
     const saveColumnName = editColumn.relationship?.foreignKey
       ? editColumn.relationship?.foreignKey
       : editColumn.path;
-    await mutate({
+
+    const variables = {
       id: recordId,
       updates: {
         [saveColumnName]: value,
       },
-    });
+    };
+
+    if (context?.type === "note") {
+      await mutate(
+        mutationVariables ? mutationVariables(variables) : variables,
+        { skip_updated_by_setter: true }
+      );
```

Comment:
if i'm following correctly, it looks like the only reason we need to add this`context` and `mutationVariables` override is to skip the updated by setter. maybe we should just pass that setting specifically as a prop? and i am happy to branch off right quick and add an `updated_by` field to the notes table so that it can live in harmony with our other related tables without this workaround.